tensorboard/program.py

```diff
@@ -200,6 +201,8 @@ def main(self, ignored_argv=('',)):
 
     :rtype: int
     """
+    self._install_signal_handler("SIGTERM")
+    self._install_signal_handler("SIGQUIT")
```

Comment:
I would leave out SIGQUIT.  At least to me SIGQUIT is like "hard quit" and I would expect the program to exit as promptly as possible and not really handle the signal unless it's really necessary, and our atexits are mostly for tidiness.

See e.g. here where it suggests that SIGQUIT handling should actually avoid cleaning up temporary files (which is what our atexits do):
https://www.gnu.org/software/libc/manual/html_node/Termination-Signals.html

Reply:
Interesting. Can do. The downside, of course, is that it produces a more
plausible vector by which .tensorboard-info files may become stale. But
I suppose that that’s okay; it just means that there’s more impetus to
add liveness checking where appropriate.

TIL; I guess I should hit Ctrl-Backslash less often. :-)

Reply:
Heh.  I usually at least give the program a chance to handle Ctrl+C first :)